pkg/config/config.go

```diff
@@ -56,6 +59,14 @@ func LoadFromFile(filename string) (*Config, error) {
 
 	config := &Config{Filename: filename}
 
+	// We don't want a dependency to root command here
+	str, ok := os.LookupEnv("GCTL_LINK_KUBECONFIG")
+	if ok {
+		if val, err := strconv.ParseBool(str); err == nil {
```

Comment:
shouldn't we let the user know if `GCTL_LINK_KUBECONFIG` could not be parsed to bool indicating a wrong value or just silently ignore?